content/docs/for-buildpack-authors/concepts/build-plan.md

```diff
@@ -3,8 +3,42 @@ title="What is the build plan?"
 weight=99
 +++
 
+A **build plan** is a `toml` file is the output of the [detect](https://buildpacks.io/docs/for-buildpack-authors/concepts/lifecycle-phases/#phase-2-detect) phase, in which each image extension or component buildpack may express the dependencies it requires and the dependencies it provides.
+
```

Comment:
We probably need to describe here that the build plan is used to figure out the order of the buildpacks in the build phase.  As an example, a `maven` buildpack that detects a `pom.xml` could _require_ that a `"jdk"` is provided.  The `provides` are specified by individual buildpacks.  That is to say, if I wrote a buildpack that installs a `NodeJS` runtime then it would label the provider as `"node"`, or `"nodejs"` or some other string.  To `require` that buildpack you would have to read the documentation to find our what string is uses as a label for the provider.

Reply:
Isn't this already added under https://buildpacks.io/docs/for-buildpack-authors/how-to/write-buildpacks/use-build-plan/?

Reply:
Who decides this? We need to discuss this in more details. @natalieparellano

Reply:
That makes sense to me.

Reply:
We will solve this problem by linking

Reply:
linked the two pages